python/caffe/_caffe.cpp

```diff
@@ -490,7 +490,8 @@ BOOST_PYTHON_MODULE(_caffe) {
   bp::class_<SolverParameter>("SolverParameter", bp::no_init)
     .add_property("max_iter", &SolverParameter::max_iter)
     .add_property("display", &SolverParameter::display)
-    .add_property("layer_wise_reduce", &SolverParameter::layer_wise_reduce);
+    .add_property("layer_wise_reduce", &SolverParameter::layer_wise_reduce)
+    .add_property("base_lr", &SolverParameter::base_lr, &SolverParameter::set_base_lr);
```

Comment:
This change is included in #6209 which will be merged prior to any other changes to `_caffe.cpp`

Reply:
Oho, looking at #6209 looks like everything in this pull request is done also there, no?